flink-streaming-connectors/flink-connector-cassandra/src/main/java/org/apache/flink/streaming/connectors/cassandra/CassandraTupleWriteAheadSink.java

```diff
@@ -49,14 +49,8 @@
 	private final String insertQuery;
```

Comment:
The `CassandraTupleWriteAheadSink` and the `GenericWriteAheadSink` don't define a `serialVersionUID`.